Looks like a Jenkins environment issue in the latest tests.
```
18:36:01 Caused by: java.lang.OutOfMemoryError: unable to create new native thread
```
I will re-trigger the tests.

--- a/generator/src/main/java/org/apache/kafka/message/IsNullConditional.java
+++ b/generator/src/main/java/org/apache/kafka/message/IsNullConditional.java
@@ -21,7 +21,7 @@ package org.apache.kafka.message;
  * Creates an if statement based on whether or not a particular field is null.
  */
 public final class IsNullConditional {
-    static IsNullConditional forField(String name) {
+    static IsNullConditional forName(String name) {
         return new IsNullConditional(name);
     }
 

--- a/generator/src/test/java/org/apache/kafka/message/IsNullConditionalTest.java
+++ b/generator/src/test/java/org/apache/kafka/message/IsNullConditionalTest.java
@@ -29,7 +29,7 @@ public class IsNullConditionalTest {
     public void testNullCheck() throws Exception {
         CodeBuffer buffer = new CodeBuffer();
         IsNullConditional.
-            forField("foobar").
+            forName("foobar").
             nullableVersions(Versions.parse("2+", null)).
             possibleVersions(Versions.parse("0+", null)).
             ifNull(() -> {
@@ -46,7 +46,7 @@ public class IsNullConditionalTest {
     public void testAnotherNullCheck() throws Exception {
         CodeBuffer buffer = new CodeBuffer();
         IsNullConditional.
-            forField("foobar").
+            forName("foobar").
             nullableVersions(Versions.parse("0+", null)).
             possibleVersions(Versions.parse("2+", null)).
             ifNull(() -> {
@@ -68,7 +68,7 @@ public class IsNullConditionalTest {
     public void testNotNullCheck() throws Exception {
         CodeBuffer buffer = new CodeBuffer();
         IsNullConditional.
-            forField("foobar").
+            forName("foobar").
             nullableVersions(Versions.parse("0+", null)).
             possibleVersions(Versions.parse("2+", null)).
             ifNotNull(() -> {
@@ -85,7 +85,7 @@ public class IsNullConditionalTest {
     public void testNeverNull() throws Exception {
         CodeBuffer buffer = new CodeBuffer();
         IsNullConditional.
-            forField("baz").
+            forName("baz").
             nullableVersions(Versions.parse("0-2", null)).
             possibleVersions(Versions.parse("3+", null)).
             ifNull(() -> {
@@ -103,7 +103,7 @@ public class IsNullConditionalTest {
     public void testNeverNullWithBlockScope() throws Exception {
         CodeBuffer buffer = new CodeBuffer();
         IsNullConditional.
-            forField("baz").
+            forName("baz").
             nullableVersions(Versions.parse("0-2", null)).
             possibleVersions(Versions.parse("3+", null)).
             ifNull(() -> {

